examples/build_sample_experiment.py

```diff
@@ -135,7 +145,15 @@ def build_image(fov_count, hyb_count, ch_count, z_count, default_shape=(1536, 10
     return collection
 
 
-def write_experiment_json(path, fov_count, hyb_dimensions, aux_name_to_dimensions):
+def write_experiment_json(
```

Comment:
Ohhhh, I think I finally get where this is all going. If you have a pile of images with correct names, are you working towards something that will build the `experiment.json` for a user?